tools/offcputime.py

```diff
@@ -96,12 +96,11 @@ def signal_ignore(signal, frame):
 BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
 
 int oncpu(struct pt_regs *ctx, struct task_struct *prev) {
-    u32 pid;
+    u32 pid = prev->pid;
```

Comment:
shouldn't it be 'pid = bpf_get_current_pid_tgid();' instead?
moving line 109 into here...